### DISCUSSION TITLE
Independent power supply

### DISCUSSION THREAD

**Discussion by z8manu8z** [mcguirepr89/BirdNET-Pi#1141] [Q&A]
Hello everyone,

Sorry for my broken English, I'm French ;-)

I am very interested in the BirdNet-PI project, and I would like to implement it for an ornithological project.

However, I have one (small) constraint: the device will be installed in the middle of the forest and must therefore have an independent power supply.

Have any of you already done such an installation? What do you recommend ?

Thanks a lot for your help !

Manu

**Comment by tphakala** [comment]
I think BirdNET-Pi is wrong tool for your use case, Audiomoth audio recorder would be more suitable https://groupgets.com/products/audiomoth-v1-2-0-group-order-round-23

You just need to run post recording analysis for Audiomoth recordings using BirdNET Analyzer or Chirpity (https://github.com/Mattk70/Chirpity-Electron)

**Reply by z8manu8z** [reply]
Thanks i will have a look at this project !

**Comment by pddpauw** [comment]
power consumption is approx 5W, so you can easily calculate... With a 12v 100Ah battery, you could theoretically get to 10 days (you need 12vdc to 5vdc converter). 
Alternatively you have a solar panel which you could put outside the forest (and battery closer to the Pi).  Have discribed this setup on my wiki.

Good luck!

**Reply by z8manu8z** [reply]
Thanks a lot i'll look at this solution ;-)

**Comment by jalbarra** [comment]
Hi,

We set up a station with the configuration that I attached and it has been
working all winter without problems.
The design allows data to be consulted remotely, only when there is sun and
the Pb battery has a minimum charge, and the records are made day and
night, as long as the second lithium battery system has a charge. The
Pijuice hat manages energy flows, and turns the RBPY on/off in an orderly
manner.
I hope it is useful to you

Greetings,

**Reply by z8manu8z** [reply]
Hi,

It's very interesting thanks ! But i can't see any file attached ?
